lib/release_page.dart

```diff
@@ -50,6 +50,9 @@ class ReleasePage extends StatelessWidget {
         selected?.version,
       );
     }
+    if (!device.flags.contains(FwupdDeviceFlag.usableDuringUpdate)) {
+      dialogText += ' ${l10n.deviceUnavailable}';
```

Comment:
can we pass an optional description as a separate argument to `showConfirmationDialog` and show it on a separate line with a smaller and lighter font?

![Screenshot from 2022-09-21 13-25-40](https://user-images.githubusercontent.com/140617/191493040-197d77ee-1135-4ba7-adda-a2cc67b57d02.png)

Reply:
Yes, we can!